README.md

```diff
@@ -56,3 +56,11 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Testing
+A number of unit tests have been added in /tests folder. They have been written with pytest in mind.
+Before unit testing make sure to have a task recently completed to be able to trigger all the tests.
+
+### How to run the tests
+Using VSCode add [Test Explorer UI extension](https://marketplace.visualstudio.com/items?itemName=hbenl.vscode-test-explorer) and [Python Test Explorer for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=LittleFoxTeam.vscode-python-test-adapter). Navigate to extension GUI and execute tests.
+
```

Comment:
You usually want to be able to run the tests from command line as well (either for convenience or for the purposes of an automated pipeline later).

`poetry run pytest`